File: C#FundamentalsSeptember2016/TechnologyFundamentals/ProgrammingFundamentals/5.ArraysAndLists/DayOfWeek/DayOfWeek.cs
using System;

namespace DayOfWeek
{
    public class DayOfWeek
    {
        public static void Main(string[] args)
        {
            string[] days = { "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" };
            int choose = int.Parse(Console.ReadLine());
            if (choose >= 1 && choose <= 7)
            {
                Console.WriteLine(days[choose - 1]);
            }
            else
            {
                Console.WriteLine("Invalid Day!");
            }
        }
    }
}

File: JSFundamentalsMay2016/03.Functions/DayOfWeek.js
function dayOfWeek(input) {
    let days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
    let result = days.indexOf(input) + 1;

    return result != 0 ? result : 'error';
}

console.log(dayOfWeek('Friday'));